packages/appstore/src/types/common.types.ts

```diff
@@ -183,3 +184,5 @@ export type TLinkedTo = {
     balance?: string;
     currency?: string;
 };
+
+export type TWalletAccount = NonNullable<ReturnType<typeof useWalletList>['data']>[number];
```

Comment:
@thisyahlen-deriv Won't recommend putting types in a global file and importing them around, Better to isolate each component prop type.

We can move forward for now, You can ignore this comment 🙇🏻